modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -0,0 +1,54 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Nícolas Moura
+ */
+public class UpgradeCatchAllCheck {
+
+	public void method() {
+		HtmlUtil.extractText(testEntry.getContent());
+
+		if (true) {
+			String contentVariableLineBreak = HtmlUtil.extractText(
+				testEntry.getContent());
+		}
+	}
+
+	public void method(CommerceAddress commerceAddress) {
+		Region region = commerceAddress.getCountry();
+
+		region.getCode();
+
+		commerceAddress.getRegion().getCode();
+
+		commerceAddress.getRegion(
+		).getCode();
```

Comment:
@nicolas-sss, Brian probably will complain about it since it's not following Liferay Code Convention and it's not in the ungly/stange bucket mentioned by him. Can you please remove this part?

Reply:
This isn't the ugly part, this is about the pattern, the pattern is to test the chaining in this case.